### ISSUE TITLE
nvBWT (and nvSetBWT) do not deal with multifasta (treats only first ref into)

### ISSUE THREAD

**Issue by FTouzain** [NVlabs/nvbio#44]
Hello,

When __indexing a multifasta__ file, both __nvBWT and nvSetBWT__ work, but they __treat only the first sequence__ in my multifasta file.
(nvBowtie works also, treating this only first reference sequence).

How to deal with multifasta (therefore several fasta sequences in a same file), please?
Thank you in advance.

**Comment by dnbuckley** [comment]
I also have this issue.

**Comment by andreirajkovic** [comment]
Any updates on this?

**Comment by andrewmoeller** [comment]
I agree that it would be great if these tools were applicable to multifasta files, particularly because it would allow the tools to be used in metagenomics studies.
One area where it could be particularly helpful is assembling genomes from metagenomes.  The current approaches can incorporate information about the relative abundances of contigs across samples into the binning process, but typically this is only done for a small number of samples due to reliance on cpu-based mapping software. Extending the nv tools to work with multifastas could be a huge advance!
